components/provisioner/internal/api/resolver_integration_with_gardener_test.go

```diff
@@ -259,14 +255,11 @@ func TestProvisioning_ProvisionRuntimeWithDatabase(t *testing.T) {
 
 			resolver := api.NewResolver(provisioningService, validator, tenantUpdater)
 
-			err = insertDummyReleaseIfNotExist(releaseRepository, uuidGenerator.New(), kymaVersion)
-			require.NoError(t, err)
-
-			fullConfig := gqlschema.ProvisionRuntimeInput{RuntimeInput: &runtimeInput, ClusterConfig: &clusterConfig, KymaConfig: kymaConfig}
+			fullConfig := gqlschema.ProvisionRuntimeInput{RuntimeInput: &runtimeInput, ClusterConfig: &clusterConfig}
 
 			testProvisionRuntime(t, ctx, resolver, fullConfig, config.runtimeID, shootInterface, secretsInterface, config.auditLogTenant)
 
-			testUpgradeRuntimeAndRollback(t, ctx, resolver, dbsFactory, config.runtimeID)
+			//testUpgradeRuntimeAndRollback(t, ctx, resolver, dbsFactory, config.runtimeID)
```

Comment:
Remove, don't comment out
We have `git` if we need it back

Reply:
Ok, I will delete this.